src/ex1/style.css

```diff
@@ -0,0 +1,96 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  font-family: Arial, Helvetica, sans-serif;
+  margin: 0;
+}
+
+button {
```

Comment:
changing the native element's style directly is considered a bad practice as it affects all buttons, but you might want a few different button styles - this will require you to keep overriding styles you don't want every time you change this style. better use base class name and use it across all the buttons